js/custom_projection.js

```diff
@@ -0,0 +1,147 @@
+// A modified d3.geo.albersUsa to include Puerto Rico.
+function albersUsaPr() {
```

Comment:
this function name seems misleading since it is adding alaska and hawaii, right?

Reply:
I guess I don't see the misleading as Alaska and Hawaii are apart of the USA, what would you rather have it?  A simple change is always easy.

Reply:
"a part" vs "apart" ...it's only now dawned on my that the space can make a pretty significant difference :smile_cat:

Reply:
I'm looking at this a second time .... it is less misleading now that I read it again. So nevermind :)